ui/analyse/src/study/studyChapters.ts

```diff
@@ -95,17 +96,13 @@ export function view(ctrl: StudyCtrl): VNode {
     }
     vData.count = newCount;
     if (canContribute && newCount > 1 && !vData.sortable) {
-      const makeSortable = function () {
-        vData.sortable = window.Sortable.create(el, {
-          draggable: '.draggable',
-          handle: 'ontouchstart' in window ? 'span' : undefined,
-          onSort() {
-            ctrl.chapters.sort(vData.sortable.toArray());
-          },
-        });
-      };
-      if (window.Sortable) makeSortable();
-      else lichess.loadScript('javascripts/vendor/Sortable.min.js').then(makeSortable);
```

Comment:
it's lazy loaded because only a fraction of study users can sort the chapters - and only a fraction of them will actually do it.